src/System.Linq/src/System/Linq/Utilities.cs

```diff
@@ -3,11 +3,12 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Collections.Generic;
+using System.Diagnostics;
 
 namespace System.Linq
 {
     /// <summary>
-    /// Contains helper methods for System.Linq. Please put enumerable-related methods in <see cref="EnumerableHelpers"/>.
+    /// Contains helper methods for System.Linq. Please put enumerable-related methods in <see cref="EnumerableHelpers"/> if possible.
```

Comment:
Self-note: The helper function I introduced to this class could probably be put in `EnumerableHelpers`; that class would probably just have to be made partial and have a separate file only included in Linq.